oci_runtime.rst

```diff
@@ -31,168 +31,203 @@ tools from the OCI ecosystem.
+    mounted as read-only).
+
+* ``--cwd`` (synonym: ``--pwd``) to set a custom starting working-directory for
+  the container.
+
+* ``--home`` to set the in-container user's home directory. Supplying a single
+  location (e.g. ``--home /myhomedir``) will result in a new tmpfs directory
+  being created at the specified location inside the container, and that dir
+  being set as the in-container user's home dir. Supplying two locations
+  separated by a colon (e.g. ``--home /home/user:/myhomedir``) will result in
+  the first location on the host being bind-mounted as the second location
+  in-container, and set as the in-container user's home dir.
+
+* ``--scratch`` (shorthand: ``-S``) to mount a tmpfs scratch directory in the
+  container.
+
+* `--workdir <workdir>`: if specified, will map `/tmp` and `/var/tmp` in the
+  container to `<workdir>/tmp` and `<workdir>/var_tmp`, respectively, on the
+  host (rather than to tmpfs storage, which is the default). If `--scratch
+  <scratchdir>` is used in conjunction with `--workdir`, scratch directories
+  will be mapped to subdirectories nested under `<workdir>/scratch` on the host,
+  rather than to tmpfs storage.
+
+* ``--no-home`` to prevent the container home directory from being mounted.
+
+* ``--no-mount`` to disable the mounting of ``proc``, ``sys``, ``devpts``,
+  ``tmp``, and ``home`` mounts in the container. Note: ``dev`` cannot be
+  disabled in OCI-mode, and ``bind-path`` mounts are not supported.
+
+* Support for the ``SINGULARITY_CONTAINLIBS`` environment variable, to specify
+  libraries to bind into ``/.singularity.d/libs/`` in the container.
+
+* ``--hostname`` to set a custom hostname inside the container. (This requires a
+  UTS namespace, therefore this flag will infer ``--uts``.)
+
+* Handling ``--dns`` and ``resolv.conf`` on a par with native mode: the
+  ``--dns`` flag can be used to pass a comma-separated list of DNS servers that
+  will be used in the container; if this flag is not used, the container will
+  use the same ``resolv.conf`` settings as the host.
+
+* Additional namespace requests with ``--net``, ``--uts``, ``--user``.
+
+* ``--no-privs`` to drop all capabilities from the container process and enable
+  the ``NoNewPrivileges`` flag.
+
+* ``--keep-privs`` to keep effective capabilities for the container process
+  (bounding set only for non-root container users).
+
+* ``--add-caps`` and ``--drop-caps``, to modify capabilities of the container
+  process.
+
 * ``--rocm`` to bind ROCm GPU libraries and devices into the container.
-* ``--nv`` to bind Nvidia driver / basic CUDA libraries and devices into the container.
+
+* ``--nv`` to bind NVIDIA driver / basic CUDA libraries and devices into the
+  container.
+
 * ``--apply-cgroups``, and the ``--cpu*``, ``--blkio*``, ``--memory*``,
   ``--pids-limit`` flags to apply resource limits.
 
-Other features are not supported, including but not limited to:
+Features that are not yet supported include but not limited to:
```

Comment:
Would be good to cross-check with https://github.com/sylabs/singularity/blob/main/internal/pkg/runtime/launcher/oci/launcher_linux.go#L102 and remove the `not limited to` at this point.

Reply:
Makes sense – but I think you and I would need to sit down & talk this through, to make sure we're on the same page about each thing there. What I mean is: not everything listed there as a `BadOpt` qualifies as a feature that is "not supported"; some (e.g. `Writable`) are just cases where things work a little differently in OCI mode, and so the flag doesn't make sense (and thus, it would not be appropriate to list it under "Other features [that] are not supported").

So we should discuss the distillation of the the contents of that method into an actual list to be included in the docs, I think.